package.json

```diff
@@ -29,12 +29,13 @@
   "homepage": "https://github.com/stampit-org/stamp-specification#readme",
   "devDependencies": {
     "babel-cli": "^6.6.5",
-    "babel-eslint": "^5.0.0",
+    "babel-eslint": "^6.1.2",
     "babel-polyfill": "^6.7.2",
     "babel-preset-es2015": "^6.6.0",
     "babel-register": "^6.7.2",
-    "check-compose": "^2.0.0",
-    "eslint": "~2.2.0"
+    "check-compose": "^3.1.2",
+    "eslint": "^2.13.1",
```

Comment:
I believe this repo can drop support for anything. This is just a sample implementation. We do not care of it cross compatibility.

I can migrate this PR to use the latest eslint if you insist.

I put eslint changes to this PR because I had them laying on my hdd for some time. It was simpler that way. That's why it is eslint@2. Surely I can bump it to eslint@3. Should I do it in this PR?